packages/info/logger/mixin.server.js

```diff
@@ -0,0 +1,35 @@
+const { sync } = require('mixinable');
+const { Mixin } = require('hops-mixin');
+const { internal: bootstrap } = require('hops-bootstrap');
+const { createLogger } = require('.');
+
+const { callable } = sync;
+const { validate, invariant } = bootstrap;
+
+class ServerLogMixin extends Mixin {
```

Comment:
Could you not just pass the already configured logger through?
Use the core mixin to register a middleware and add the logger to res.locals and then in the server mixin you use bootstrap to get the logger out of res.locals and have a `getLogger` hook that returns the logger or undefined (when called before `bootstrap`.